src/model/actions/quality-action.ts

```diff
@@ -40,7 +40,7 @@ export abstract class QualityAction extends GeneralAction {
       buffMult += 0.5;
     }
 
-    buffMod = Math.fround(buffMod) * Math.fround(buffMult);
+    buffMod = Math.fround((buffMod * buffMult * (100 + iqMod * 10)) / 100);
```

Comment:
I'm not sure what is the intent behind initial value of buffMod, it's always 1 as far as I can see (getBaseBonus).

buffMult is always a power-of-two, so there is no sense in rounding it - it's always represented exactly as any floating-point format.

This change does not break any existing testcases, however obviously it could break something else...

Reply:
iirc it was because we used to calculate more stuff in this, the simulator exists since several years now so it could easily be a relicate of this.